src/main.rs

```diff
@@ -142,7 +142,7 @@ fn cli() -> Command {
         ))
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
```

Comment:
Is there a reason we need to make the tokio runtime for the client single threaded?

Reply:
Fraser and I cleaned up imports of tokio, removing necessary mutli thread rt-multi-thread to rt and the compiler asked to add that attribute.  

```
   Compiling casper-client v2.0.0 (/casper/casper-client-rs)
error: The default runtime flavor is `multi_thread`, but the `rt-multi-thread` feature is disabled.
   --> src/main.rs:145:1
    |
145 | #[tokio::main]
    | ^^^^^^^^^^^^^^
    |
    = note: this error originates in the attribute macro `tokio::main` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `casper-client` (bin "casper-client") due to previous error
```

Reply:
@Fraser999 Can you please confirm we want to get rid of rt-multi-thread in the client ? thanks